Unused variables

```
➜ make
make -C /lib/modules/5.5.2-arch2-2/build M=/tmp/xmm7360-pci modules
make[1]: Entering directory '/usr/lib/modules/5.5.2-arch2-2/build'
  CC [M]  /tmp/xmm7360-pci/xmm7360.o
/tmp/xmm7360-pci/xmm7360.c: In function ‘xmm7360_remove’:
/tmp/xmm7360-pci/xmm7360.c:1177:6: warning: unused variable ‘i’ [-Wunused-variable]
 1177 |  int i;
      |      ^
/tmp/xmm7360-pci/xmm7360.c: In function ‘xmm7360_probe’:
/tmp/xmm7360-pci/xmm7360.c:1375:6: warning: unused variable ‘i’ [-Wunused-variable]
 1375 |  int i, ret;
      |      ^
  Building modules, stage 2.
  MODPOST 1 modules
  CC [M]  /tmp/xmm7360-pci/xmm7360.mod.o
  LD [M]  /tmp/xmm7360-pci/xmm7360.ko
make[1]: Leaving directory '/usr/lib/modules/5.5.2-arch2-2/build'
```

--- a/xmm7360.c
+++ b/xmm7360.c
@@ -1174,7 +1174,6 @@ static void xmm7360_dev_deinit(struct xmm_dev *xmm)
 static void xmm7360_remove(struct pci_dev *dev)
 {
 	struct xmm_dev *xmm = pci_get_drvdata(dev);
-	int i;
 
 	xmm7360_dev_deinit(xmm);
 
@@ -1372,7 +1371,7 @@ void xmm7360_dev_init_work(struct work_struct *work)
 static int xmm7360_probe(struct pci_dev *dev, const struct pci_device_id *id)
 {
 	struct xmm_dev *xmm = kzalloc(sizeof(struct xmm_dev), GFP_KERNEL);
-	int i, ret;
+	int ret;
 
 	xmm->pci_dev = dev;
 	xmm->dev = &dev->dev;

